lte/gateway/c/core/oai/tasks/nas/emm/emm_headers.hpp

```diff
@@ -22,12 +22,18 @@ limitations under the License.
 #include <sys/types.h>
 
 #include "lte/gateway/c/core/common/common_defs.h"
-#include "lte/gateway/c/core/oai/include/nas/securityDef.h"
+#include "lte/gateway/c/core/oai/include/nas/securityDef.hpp"
```

Comment:
Can we use 
```cpp
#pragma once
```
?

Reply:
This is a temporary file and will be removed in the next PR